apps/quickbooks_online/utils.py

```
@@ -150,10 +159,31 @@ def get_tax_inclusive_amount(self, amount, default_tax_code_id):
+        """
+        Checks if the sync is allowed
+
+        Returns:
+            bool: True
+        """
+        if attribute_count > SYNC_UPPER_LIMIT[attribute_type]:
+            workspace_created_at = Workspace.objects.get(id=self.workspace_id).created_at
+            if workspace_created_at > timezone.make_aware(datetime(2024, 10, 1), timezone.get_current_timezone()):
+                return False
+            else:
+                return True
```

Ruff

Return the negated condition directly

Inline condition

(SIM103)